ProcessMaker/processmaker
# Implement ForUserScope into develop

## Issue & Reproduction Steps
Some code got missed in the merge from 7193. See [7325 issue](https://processmaker.atlassian.net/browse/FOUR-7325) issue for details.

## Solution
- Re-merged the branch into this branch (based off develop)

## How to Test
- Delete the table request_user_permissions on your local instance
- Create a non-admin user and give them permission to start a simple process
- Log in as the non-admin users and start the process
- Make sure the process is visible on the requests page

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-7325

## Code Review Checklist

..
